packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -203,30 +122,101 @@ function MyApp() {
 function MyComponent() {
   const breakpoints = useBreakpoints();
   // here we can use useBreakpoints to access the value from the ToolkitProvider prop
-  return <div>Hello World!</div>;
+  return <div>Breakpoints: {breakpoints}</div>;
 }
 ```
 
-<!-- #### Portal
+## AriaAnnouncer
 
-Note: `Portal` is not yet ready for release, the notes below are placeholder only. Need something to position the portal example
+`AriaAnnouncer` will be injected automatically inside the root element when ToolkitProvider is wrapping the app. If you use
+nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
```

Comment:
```suggestion
nested ToolkitProviders, these will not inject further instances of `AriaAnnouncer`.
```
Have to figure out what we do with `ToolkitProviders` here does it get back ticks or is the s excluded.